pdc/apps/common/renderers.py

```diff
@@ -196,6 +196,7 @@ def replace_url(match):
     'BooleanFilter': 'bool',
     'CaseInsensitiveBooleanFilter': 'bool',
     'ActiveReleasesFilter': 'bool',
+    'NumberFilter': 'int',
```

Comment:
Where is 'NumberFilter' used? Seems we are using MultiIntFilter() in following file

Reply:
That's the default filter type used by Django for the "id" fields.

Reply:
Is there any issues if we don't add this line? Can you give an example?
For most of the endpoint we can use the retrieve, do you mean that id?

Reply:
> Is there any issues if we don't add this line?

Without this line, API docs show that the "id" is "string" in list of filters ("Query params" section in docs). This PR fixes just the docs for filters.

Reply:
ok, got it.